<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QWidget" name="Profile">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #171A21;</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>622</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollArea
{
	border: none;
}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1254</width>
      <height>622</height>
     </rect>
    </property>
    <widget class="QWidget" name="profileCard" native="true">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>0</y>
       <width>1111</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
	/*background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                stop:0 #2a475e, stop: 0.2 rgba(27, 50, 56, 0.9),
                stop:0.9 rgba(27, 50, 56, 0.9))*/
	background-image: url('D:/fucking uni/DB/project/profile_header_bg_texture.jpg');
}</string>
     </property>
     <widget class="QLabel" name="levelNumberLabel">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>60</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	width:28px;
	height:28px;
    border-width: 3px;        
  	border-style: solid;
	border-color: gray;
  	border-radius: 16px;
	font-size:15px;
	color:white;
	background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QLabel" name="profileImg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>profileImg</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	font-size: 30px;
	color:white;
	background: none;
}</string>
      </property>
      <property name="text">
       <string>ddssee71</string>
      </property>
     </widget>
     <widget class="QLabel" name="bioExLabel">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	font-size: 15px;
	padding-top: 1px;
	color:gray;
	background: none;
}</string>
      </property>
      <property name="text">
       <string>skfsljfsjflsjdflksaaaaaaaaaaaaaaaaaaaaa</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelBio">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	font-size: 20px;
	color:white;
	background: none;
}</string>
      </property>
      <property name="text">
       <string>Bio:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLevel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	font-size: 30px;
	color:white;
	background: none;
}</string>
      </property>
      <property name="text">
       <string>Level</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEditProfile">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>140</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	color:white;
	background: transparent;
	border: 1px solid white;
}</string>
      </property>
      <property name="text">
       <string>Edit Profile</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>221</y>
       <width>1091</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
	background-color: #0f101a;
}</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>711</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	font-size: 25px;
	color: white;
	background-color: #000000;
}</string>
      </property>
      <property name="text">
       <string>Recent Activity</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>20</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color: gray;
	font-size: 22px;
}</string>
      </property>
      <property name="text">
       <string>Currently Offline</string>
      </property>
     </widget>
     <widget class="QLabel" name="lastOnlineLabel">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>50</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color: gray;
	font-size: 15px;
}</string>
      </property>
      <property name="text">
       <string>Last Online 4 hrs, 56 mins ago</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gamesPushButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>100</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	color: white;
	font-size: 20px;
	border: none;
	background-color: transparent;
}
QPushButton:hover
{
	text-decoration: underline;
}</string>
      </property>
      <property name="text">
       <string>Games</string>
      </property>
     </widget>
     <widget class="QPushButton" name="achievementsPushButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>140</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	color: white;
	font-size: 20px;
	border: none;
	background-color: transparent;
}
QPushButton:hover
{
	text-decoration: underline;
}</string>
      </property>
      <property name="text">
       <string>Achievements</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listGames">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>711</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget
{
	border:none;
}
QListWidget::item
 {
 	margin-bottom: 10px;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="friendPushButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>180</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	color: white;
	font-size: 20px;
	border: none;
	background-color: transparent;
}
QPushButton:hover
{
	text-decoration: underline;
}</string>
      </property>
      <property name="text">
       <string>Friends</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listFriend">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>220</y>
        <width>256</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget
{
	border:none;
}
QListWidget::item
 {
 	margin-bottom: 10px;
}</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
